README.md

```diff
@@ -1,17 +1,44 @@
 # Sawtooth Simple Supply
 
+Sawtooth Simple Supply is a basic example of a distributed supply-chain
+application. It provides asset transfer with location tracking, which allows
+users to transfer assets and track them as they move through a supply chain.
+
+Sawtooth Simple Supply is built on Hyperledger Sawtooth, an enterprise
+blockchain platform. To learn more about Hyperledger Sawtooth, see the
+[sawtooth-core repository](https://github.com/hyperledger/sawtooth-core) and the
+[Hyperledger Sawtooth documentation](https://sawtooth.hyperledger.org/docs/).
+
+The Simple Supply components in this repository include:
+
+- A transaction processor that handles the Simple Supply transaction logic
+
+- A custom REST API that provides HTTP/JSON endpoints for querying blockchain
+  data, handles batches and transactions, and manages user information
```

Comment:
I'd recommend:

> data, **handles** batches and transactions, and manages user information

Reply:
I don't see a difference.  Should the verbs change to match "querying" instead of "provides"? That is, should this be:

... for querying blockchain data, handling batches and transactions, and managing user information...

The difference is whether this sentence means that the REST API provides these three things (so verbs would match "provides") or the endpoints do (in which case, the verbs would match "querying").